tests/test_symbolics.py

```diff
@@ -18,3 +20,33 @@ def test_float_indices():
     indices = u.subs({x: 1.0}).indexify().indices[0]
     assert len(indices.atoms(sympy.Float)) == 0
     assert indices == 1
+
+
+@pytest.mark.parametrize('so', [2, 4])
+def test_solve(so):
+    """
+    Test that our solve produces the correct output and in a smaller time
```

Comment:
Isn't this better:

...and in less time than ("here goes solver type before solve").... (?)